src/figuresClasses.ts

```diff
@@ -1,11 +1,87 @@
-export interface Figure {}
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
+
+  color: Color;
+
+  constructor(
+    color: Color,
+    private a: number,
+    private b: number,
+    private c: number,
```

Comment:
[CODE STYLE] - The `getArea` method of the `Triangle` class should not accept any parameters as it can use `this.a`, `this.b`, and `this.c` directly.